Exiled.API/Interfaces/IConfig.cs

```diff
@@ -15,6 +17,7 @@ public interface IConfig
         /// <summary>
         /// Gets or sets a value indicating whether the plugin is enabled or not.
         /// </summary>
+        [Description("Indicates whether the plugin is enabled or not")]
```

Comment:
The "problem" with putting the DescriptionAttribute here is that it won't be auto-generated inside configs by YamlDotNet.

Reply:
Yes, wouldn't it be better to supply it?

Reply:
I'll remove it. I thought it can get inherit attributes.